core/src/components/input/test/clear-on-edit/input.e2e.ts

```diff
@@ -1,6 +1,8 @@
 import { expect } from '@playwright/test';
 import { test, configs } from '@utils/test/playwright';
 
+const IGNORED_KEYS = ['Enter', 'Tab', 'Shift', 'Meta', 'Alt', 'Control'];
```

Comment:
I could export this array from each component file and then import it in the test file, but wasn't sure if we were comfortable with that level of coupling. For example, if that array ever gets moved/deleted (even if the underlying functionality remains) these tests will start to fail.

Reply:
True, but we also run into the potential issue of if the array is updated in the components then we have to remember to update it in this file. It might be best to export it since we can guarantee the right keys. The tests may fail but at least it would be an easy fix vs not testing the correct array.

Reply:
Wouldn't that be expected anyways? i.e. if you make a change/fix a bug I'd expect a new test to be written.

Reply:
Good point!

Reply:
I vote for leaving it as is.